device/cuda/src/cca/component_connection.cu

```diff
@@ -743,19 +742,19 @@ component_connection::output_type component_connection::operator()(
      * more efficiently. This removes the hierarchy level that
      * references to the cell module.
      */
-    vecmem::vector<channel_id> channel0(&mem);
+    std::vector<channel_id> channel0;
```

Comment:
With an interface requiring a specific "host memory resource" I'd really like these to return to being `vecmem::vector` instances.

Reply:
I'm not sure I see the point there, this is strictly host-only code that gains nothing from custom allocation schemes. And as in the other pull request, this is just data pre-processing; not performance-relevant and should be removed for production code. Not sure we should really spend much development time on that kind of code.